backend/dal/Migrations/20210122232754_v01.06.01.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Pims.Dal.Migrations
+{
+    public partial class v010601 : Migration
```

Comment:
Doesn't inherit from `SeedMigration` so no scripts will be run, as the `PreUp(...)` and `PostUp(...)` are not available unless you use the correct inheritance.  Use the `make db-add n=v01.06.01` to automate the process for you.